api/app/db/models/auto_spatial_advisory.py

```diff
@@ -137,3 +137,25 @@ class RunParameters(Base):
                       create_type=False), nullable=False, index=True)
     run_datetime = Column(TZTimeStamp, nullable=False, index=True)
     for_date = Column(Date, nullable=False, index=True)
+
+
+class AdvisoryElevationStats(Base):
+    """ 
+    Summary statistics about the elevation of area with high hfi (4k-10k and >10k) per firezone
+    based on the set run_type, for_date and run_datetime.
+    """
+    __tablename__ = 'advisory_elevation_stats'
+    __table_args__ = (
+        {
+            'comment': 'Elevation stats per fire zone by advisory threshold'
+        }
+    )
+    id = Column(Integer, primary_key=True, index=True)
+    advisory_shape_id = Column(Integer, ForeignKey('advisory_shapes.id'), nullable=False, index=True)
+    threshold = Column(Integer, ForeignKey('advisory_hfi_classification_threshold.id'), nullable=False)
+    run_parameters = Column(Integer, ForeignKey('run_parameters.id'), nullable=False, index=True)
+    minimum = Column(Float, nullable=False)
+    quartile_25 = Column(Float, nullable=False)
+    median = Column(Float, nullable=False)
+    quartile_75 = Column(Float, nullable=False)
+    maximum = Column(Float, nullable=False)
```

Comment:
shouldn't there also be a column here for `median`? The median value is calculated in auto_spatial_advisory/elevation.py

Reply:
It's hiding on line 159!